modular_skyrat/modules/death_consequences_perk/death_consequences_trauma.dm

```diff
@@ -0,0 +1,525 @@
+	/// The amount our victims crit threshold will be reduced by at [stamina_damage_max_degradation] degradation.
+	var/max_stamina_damage = 80
+
+	/// Used for updating our crit threshold reduction. We store the previous value, then subtract it from crit threshold, to get the value we had before we adjusted.
+	var/crit_threshold_currently_reduced_by = 0
+
+	/// The last world.time we sent a message to our owner reminding them of their current degradation. Used for cooldowns and such.
+	var/time_of_last_message_sent = -DEATH_CONSEQUENCES_TIME_BETWEEN_REMINDERS
+	/// The time between each reminder ([degradation_messages]).
+	var/time_between_reminders = DEATH_CONSEQUENCES_TIME_BETWEEN_REMINDERS
+
+	/// The current level of degradation. Used mostly for reminder messages.
+	var/current_degradation_level = DEGRADATION_LEVEL_NONE
+
+	// Will be iterated through sequentially, so the higher a path is, the quicker it'll be searched
+	// Make sure to put the larger bonuses and more specific types higher than the generic ones
+	/// A assoc list of (atom/movable typepath -> mult), where mult is used as a multiplier against passive living degradation reduction.
+	var/static/list/buckled_to_recovery_mult_table = list(
+		/obj/structure/bed/medical = 5,
+		/obj/structure/bed = 3,
+
+		/obj/structure/chair/comfy = 2,
+		/obj/structure/chair/sofa = 2,
+		/obj/structure/chair = 1.5,
+
+		/mob/living = 1.25, // being carried
+	)
+	/// Only used if the thing we are buckled to is not in [buckled_to_recovery_mult_table].
+	var/static/buckled_to_default_mult = 1.15
+
+	/// The random messages that will be sent to our victim if their degradation moves to a new threshold.
+	var/static/list/degradation_messages = list(
+		DEGRADATION_LEVEL_LOW = list(
+			span_warning("Your body aches a little.") = 10,
+			span_warning("You feel a little detached from yourself.") = 10,
+			span_warning("You feel a little tired.") = 10,
+		),
+		DEGRADATION_LEVEL_MEDIUM = list(
+			span_danger("Your whole body aches...") = 10,
+			span_danger("You're starting to feel disassociated from yourself...") = 10,
+			span_danger("You're having a little difficulty thinking...") = 10,
+		),
+		DEGRADATION_LEVEL_HIGH = list(
+			span_bolddanger("Your entire body throbs!") = 10,
+			span_bolddanger("You feel like you're losing your grip on yourself!") = 10,
+			span_bolddanger("Your consciousness feels as fragile as a sheet of glass!") = 10,
+			span_bolddanger("You feel exhausted in every single possible way!") = 10,
+		),
+		DEGRADATION_LEVEL_CRITICAL = list(
+			span_revenwarning("<b>Everything hurts... it hurts so bad...</b>") = 10,
+			span_revenwarning("<b>It's so hard to think... it's so hard... so hard...</b>") = 10,
+			span_revenwarning("<b>Your body feels alien, like you don't belong in it...</b>") = 10,
+			span_revenwarning("<b>...who am I?</b>") = 1,
+			span_revenwarning("<b>...where am I?</b>") = 1,
+			span_revenwarning("<b>...what am I?</b>") = 1,
+		)
```

Comment:
Why not do the formatting in another place, so it's not constantly repeated everywhere?

Reply:
whaddaya mean by formatting? the spans?